tests/test_factory_constructors.cpp

```diff
@@ -109,8 +109,7 @@ class PyTF7 : public TestFactory7 {
     PyTF7(int i) : TestFactory7(i) { alias = true; print_created(this, i); }
     PyTF7(PyTF7 &&f) : TestFactory7(std::move(f)) { print_move_created(this); }
     PyTF7(const PyTF7 &f) : TestFactory7(f) { print_copy_created(this); }
-    virtual ~PyTF7() { print_destroyed(this); }
-    int get() override { PYBIND11_OVERRIDE(int, TestFactory7, get, /*no args*/); }
```

Comment:
Even worse, sorry again, @YannickJadoul .

Reply:
Why does this not care about being deleted, actually?

Reply:
Sorry, yes, I hadn't thought about conflicts, when merging that PR. I was just happy it was ready and clicked "merge" :-/